### PR TITLE
Updating w/outcome of drud/ddev/issues/196

### PR DESCRIPTION
**Pull request by rickmanelius**
See additional commentary here (https://github.com/drud/ddev/issues/196#issuecomment-300459091) for Linux support and here (https://github.com/drud/ddev/issues/196#issuecomment-300272958) for Windows support.

## Related Issue Link(s):

#196 

## Release/Deployment notes:

No impact on code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,10 +9,18 @@ The purpose of *ddev* is to support developers with a local copy of a site for d
...
+- OS Support
+  - macOS Sierra (fully supported with automated tests)
+  - Linux
+    * Ubuntu 16.04 LTS (fully supported with automated tests)
+    * Debian Jessie (tested manually with automated tests planned)
+    * Fedora 25 (tested manually with automated tests planned)
+  - Windows 10 Pro (**experimental support!**)
+    * See [Decisions and Plan for Linux & Windows Support](https://github.com/drud/ddev/issues/196#issuecomment-300178008) for more information and the various options on getting ddev operational on Windows 10.
+
+We are open to expanding this list to include additional OSs as well as improve our existing support for the
```
Comment (tannerjfco) [inline README.md]: Seems to be missing the rest of the sentence. Do we want to explicitly open ourselves up to this though? I think we meet the largest needs at this point, if there's a pressing need for support on another OS or distribution I'm sure we'll hear about it.

Comment (rickmanelius) [inline README.md]: Good catch. It got chopped off in the copy/paste. Also, I think the point was that we wanted to be explicit about what we do support while encouraging people to speak up versus thinking "well that's all they support and that's that".

File: README.md [hunk 2]
```diff
@@ -9,10 +9,18 @@ The purpose of *ddev* is to support developers with a local copy of a site for d
...
 
-* *Currently only tested on Ubuntu, please [let us know](https://github.com/drud/ddev/issues/new) if you hit an issue on another flavor.*
+- [docker](https://www.docker.com/community-edition)
+- OS Support
+  - macOS Sierra (fully supported with automated tests)
+  - Linux
+    * Ubuntu 16.04 LTS (fully supported with automated tests)
+    * Debian Jessie (tested manually with automated tests planned)
+    * Fedora 25 (tested manually with automated tests planned)
+  - Windows 10 Pro (**experimental support!**)
```
Comment (beeradb) [inline README.md]: If we add this, do we need to specifically update the language for our install script to indicate OSX/Linux only?

Comment (rickmanelius) [inline README.md]: Added a small tweak to the install script instructions to set the scope to Linux/macOS users. Great catch!

**Comment by rickmanelius** [comment]
@rfay is out at the moment and we have 2 out of 3 reviews as approved. Committing and closing!

**Pull request merged**